kibana/tests/kibana_test.py

```diff
@@ -629,3 +629,17 @@ def test_adding_loadBalancerIP():
     r = helm_template(config)
 
     assert r["service"][name]["spec"]["loadBalancerIP"] == "12.5.11.79"
+
```

Comment:
[Black](https://black.readthedocs.io/en/stable/) formatter requires 2 empty lines between each function.

Can you run `black kibana/tests/kibana_test.py` to fix it?